### ISSUE TITLE
MilAir speed leader always starts at (0,0)

### ISSUE THREAD

**Issue by steven-omaha** [ralphieraccoon/MilSymb#16]
Consider this MWE.
```
\documentclass{standalone}
\usepackage{milsymb}
\begin{document}
\begin{tikzpicture}[]
  \MilAir[faction=friendly,main=attack,speed leader={0}{2}] (0,0);
  \MilAir[faction=friendly,main=attack,speed leader={45}{2}] (0,2);
\end{tikzpicture}
\end{document}
```
This creates the following output. The speed leader for the second node starts at (0,0), but should start at (0, 2).

![MWE](https://i.imgur.com/msQFmMn.png)

**Comment by steven-omaha** [comment]
By the way, is it intentional that the speed leader has no arrow tip?

**Comment by ralphieraccoon** [comment]
Hi Steven,

I'm currently investigating your issue. In response to your other question, speed leaders have no arrow tip in the [APP-6(C)](https://www.awl.edu.pl/images/en/APP_6_C.pdf) documentation, so it follows that standard.

**Comment by ralphieraccoon** [comment]
Hi Steven,

I've found the issue (scoping error), but you know how it goes, you investigate one bug and find a few more! I'll roll them all into an update soon. Thanks!

**Comment by ralphieraccoon** [comment]
Fixed in 6b32639.
